ansible/roles/osdsdock/scenarios/ceph_stabilize.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2019 The OpenSDS Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+# ceph_check_interval/ceph_check_count are defined in group_vars/osdsdock.yml
+- name: ceph sanity check
+  shell: |
+    INTERVAL={{ ceph_check_interval|quote }}
+    MAX_CHECK={{ ceph_check_count|quote }}
+    declare -a ceph_stat_array=()
+    i=0
+    for (( i=0; i < ${MAX_CHECK}; i++))
+    do
+      sleep ${INTERVAL}
```

Comment:
Can this sleep be avoided? Anyways if it doesn't get the proper stats, it will sleep for $INTERVAL

Reply:
OK. I'm refactoring the loop structure.